src/App/integrationTests/collectRecords/benthicLit/App.validationDuplicateModalBenthicLit.test.js

```diff
@@ -0,0 +1,718 @@
+        project_profiles: { updates: mockMermaidData.project_profiles },
+        project_sites: { updates: mockMermaidData.project_sites },
+        projects: { updates: mockMermaidData.projects },
+      }
+
+      return res(ctx.json(response))
+    }),
+  )
+
+  renderAuthenticatedOnline(
+    <App dexieCurrentUserInstance={dexieCurrentUserInstance} />,
+    {
+      initialEntries: ['/projects/5/collecting/benthiclit/70'],
+    },
+    dexiePerUserDataInstance,
+    dexieCurrentUserInstance,
+  )
+
+  await validateCollectRecordAndResolveSimilarSite()
+
+  const resolveModal = screen.getByTestId('resolve-duplicate-site')
+  const originalSite = await within(resolveModal).findByTestId('original-site')
+  const editOriginalSiteButton = await within(originalSite).findByRole('button', {
+    name: 'Edit site',
+  })
+
+  userEvent.click(editOriginalSiteButton)
+
+  await waitForElementToBeRemoved(() => screen.queryByLabelText('project pages loading indicator'))
+  const siteCPage = await screen.findByText('Site C', {
+    selector: 'h2',
+  })
+
+  expect(siteCPage).toBeInTheDocument()
+}, 50000)
+
+test('Validate Benthic LIT collect record, get site duplicate warning, show resolve button, edit duplicate site', async () => {
+  const { dexiePerUserDataInstance, dexieCurrentUserInstance } = getMockDexieInstancesAllSuccess()
+
+  mockMermaidApiAllSuccessful.use(
+    rest.post(`${apiBaseUrl}/projects/5/collectrecords/validate/`, (req, res, ctx) => {
+      return res(ctx.status(200))
+    }),
+
+    rest.post(`${apiBaseUrl}/pull/`, (req, res, ctx) => {
+      const collectRecordWithValidation = {
+        ...mockBenthicLitCollectRecords[0],
+        validations: mockSampleEventValidationObject,
+      }
+
+      const response = {
+        benthic_attributes: { updates: mockMermaidData.benthic_attributes },
+        choices: { updates: mockMermaidData.choices },
+        collect_records: { updates: [collectRecordWithValidation] },
+        fish_families: { updates: mockMermaidData.fish_families },
+        fish_genera: { updates: mockMermaidData.fish_genera },
+        fish_species: { updates: mockMermaidData.fish_species },
+        project_managements: { updates: mockMermaidData.project_managements },
+        project_profiles: { updates: mockMermaidData.project_profiles },
+        project_sites: { updates: mockMermaidData.project_sites },
+        projects: { updates: mockMermaidData.projects },
+      }
+
+      return res(ctx.json(response))
+    }),
+  )
+
+  renderAuthenticatedOnline(
+    <App dexieCurrentUserInstance={dexieCurrentUserInstance} />,
+    {
+      initialEntries: ['/projects/5/collecting/benthiclit/70'],
+    },
+    dexiePerUserDataInstance,
+    dexieCurrentUserInstance,
+  )
+
+  await validateCollectRecordAndResolveSimilarSite()
+
+  const resolveModal = screen.getByTestId('resolve-duplicate-site')
+  const duplicateSite = await within(resolveModal).findByTestId('duplicate-site')
```

Comment:
I think for these headings we can grab the element by other means. Try `.getByRole('columnheader', {name 'Duplicate site})`